core/src/test/resources/log4j.properties

```diff
@@ -18,9 +18,20 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
+log4j.appender.requestAppender=org.apache.log4j.varia.NullAppender
+
 log4j.logger.kafka=ERROR
 log4j.logger.org.apache.kafka=ERROR
 
 # zkclient can be verbose, during debugging it is common to adjust is separately
 log4j.logger.org.I0Itec.zkclient.ZkClient=WARN
 log4j.logger.org.apache.zookeeper=WARN
+
+log4j.logger.kafka.network.RequestChannel$=WARN, requestAppender
+log4j.additivity.kafka.network.RequestChannel$=false
+
+log4j.logger.kafka.network.Processor=TRACE, requestAppender
+log4j.logger.kafka.server.KafkaApis=TRACE, requestAppender
```

Comment:
This is, in fact, possible by setting the Threshold of the appender. Check out the updated version I provided that also increases the default log level. This looks the same for most users, but if you `-PshowStandardStreams=1`, you would a) catch the request logging error and b) have enough output to figure out what went wrong.